typeDefs.js

```diff
@@ -0,0 +1,30 @@
+const { gql } = require('apollo-server-express');
+
+const typeDefs = gql`
+  type JournalEntry {
+    id: ID
+    date: String
+  }
+
+  type Query {
+    hello: String
+
+    getAllJournalEntries: [JournalEntry]
+
+    getJournalEntry(id: ID): JournalEntry
+  }
+
+  input JournalEntryInput {
+    date: String
+  }
+
+  type Mutation {
+    createJournalEntry(journalEntry: JournalEntryInput): JournalEntry
+
+    deleteJournalEntry(id: ID): String
+
+    updateJournalEntry(id: ID, journalEntry: JournalEntryInput): JournalEntry
+  }
+`;
+
+module.exports = typeDefs;
```

Comment:
Would it be better to separate these into their own files (mutations, queries) or to have one per different use (user, journal entry)